configserver/src/main/java/com/yahoo/vespa/config/server/http/HttpHandler.java

```diff
@@ -51,7 +51,8 @@ public HttpResponse handle(HttpRequest request) {
         } catch (IllegalArgumentException | UnsupportedOperationException e) {
             return HttpErrorResponse.badRequest(getMessage(e, request));
         } catch (NodeAllocationException e) {
-            return HttpErrorResponse.nodeAllocationFailure(getMessage(e, request));
+            return e.retryable() ? HttpErrorResponse.nodeAllocationFailure(getMessage(e, request))
```

Comment:
Why not throw `InvalidApplicationException` instead?

Reply:
The NodeAllicationException, that is.